### ISSUE TITLE
TLS Support?

### ISSUE THREAD

**Issue by JakeIrvine** [thomasf/solarized-css#23]
Currently the produced HTML file when using the hosted version fails on Firefox, as the stylesheet is loaded over HTTP and blocked due to mixed content. Even when changing to using the HTTPS url to load the stylesheet, several fonts in the hosted stylesheet are still loaded over HTTP, causing them to be blocked.

**Comment by slackline** [comment]
Ah, I wondered why this might be why the style is applied if I view my HTML locally but not when viewing from my web-server.

**Comment by thomasf** [comment]
yeah, iut's a problem.. IIRC org-mode has support for inlining css while rendering these days so maybe not relying on external style sheets is a better option now.

**Comment by slackline** [comment]
Thanks for the suggestion/pointer I've found a possible solution on how to achieve this with a hook [here](https://stackoverflow.com/a/37132338) and will try it out.

Oh and thanks for sharing the theme, its very neat and clean.

**Comment by magni-** [comment]
Shouldn't this have been fixed by 72d01a2b362c20fd6a14cc6bcc07aaba7a31e256 ? But the issue is that the `gh_pages` branch hasn't been updated since then.
